Gemfile

```diff
@@ -5,6 +5,8 @@ gem 'dotenv-rails'
 gem 'rails', '~> 5.2'
 gem 'bootsnap', require: false
 
+gem 'mimemagic', '= 0.3.5' # https://dev.to/cseeman/what-s-up-with-mimemagic-breaking-everything-he1
```

Comment:
You should upgrade  your gem (and the Gemfile.lock) to  mimemagic (0.3.10).

Reply:
@weedySeaDragon In order to use `0.3.10` I need to install a local mime DB on my laptop.  The recommended solution is to install `shared-mime-info` via brew (on Mac).  Do we have that package installed on DI? (I assume yes)

Reply:
I think we should use upgrade Rails to 5.2.6 -- the version they just released that does not use mime-magic.  (They revised what was needed into a gem they named 'marcel' (!) ).
We'll need to update soon just to take care of the security issues anyway.  

I tried the upgrade and ran the tests and _of course_ there was a problem.  Upgrading to that version Rails also upgrades the i18n gem to v.1.8.10.  The i18n gem changed the translate method: they no longer accept an empty Hash as an argument (in preparation for ruby 3.0, in part).  The only place we pass that in is with the Cucumber features:  it is in a method in `features/support/env.rb`:  `i18n_content`  .  (Like all problems, it took hours to find and minutes to fix.)
I have a fix for it but need to put it into a PR, etc.  Will do that tomorrow.

In doing that fix, I may have found a problem with the `features/view-site-in-lang-i18n-l10n.feature`   I need to look at that again tomorrow.

I'll put the PR up tomorrow (Tuesday).  If you want to get it working in the meantime, here's the method that will change:

```
# As of version 1.8.6, I18n.translate will no longer accept a Hash as the 2nd argument.
# See I18n.translate
def i18n_content(content, translation_params = {})
  I18n.t(content, **({locale: I18n.locale}.merge(translation_params)) )
end
```

Reply:
@weedySeaDragon Yes, I had changed that method `i18n_content` in this PR because it broke for me - the 2nd argument to that method had been wrong for some time.

I was aware of the Rails upgrade that avoided the mimemagic issue, but did not want to add that to this story.  However, if you think it's a viable approach I'm all for it.